<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [Demoblaze_Suite]">
  <test thread-count="5" name="Demoblaze_Test(failed)">
    <classes>
      <class name="com.Demoblaze.TestRunner.PlacingOrder_TestRunner">
        <methods>
          <include name="login" invocation-numbers="0"/>
          <include name="setUP"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.Demoblaze.TestRunner.PlacingOrder_TestRunner -->
      <class name="com.Demoblaze.TestRunner.Login_TestRunner">
        <methods>
          <include name="setUP"/>
          <include name="login" invocation-numbers="0"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.Demoblaze.TestRunner.Login_TestRunner -->
      <class name="com.Demoblaze.TestRunner.Contact_TestRunner">
        <methods>
          <include name="setUP"/>
          <include name="login" invocation-numbers="0"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.Demoblaze.TestRunner.Contact_TestRunner -->
    </classes>
  </test> <!-- Demoblaze_Test(failed) -->
</suite> <!-- Failed suite [Demoblaze_Suite] -->
